app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+    </style>
```

Comment:
I think that you could use the "Theme.AppCompat.Light.NoActionBar" directly instead of create a new one just to inherit from an existing style but not modify it at all

Reply:
Solved!